packages/expandable-search-bar/src/expandable-search-bar.ts

```diff
@@ -80,6 +80,19 @@ export default class ExpandableSearchBar extends LitElement {
     `;
   }
 
+  /**
+   * Workaround for lit-element 2.2.1 not reflecting this.searchTerm 
```

Comment:
This isn't really a workaround so much as how form inputs work. The `searchTerm` property isn't bound to the input's value as a reactive property so you have to do set it when the value gets updated.